sql/firefox_accounts_derived/fxa_log_auth_events_v1/query.sql

```diff
@@ -0,0 +1,35 @@
+SELECT
+  jsonPayload.fields.event,
+  jsonPayload.fields.user_id,
```

Comment:
`user_id` is always null in `docker_fxa_auth_*`, so we should probably drop this field.